test/Microsoft.Azure.Devices.Edge.Test.Common/linux/OsPlatform.cs

```diff
@@ -24,7 +24,7 @@ public async Task<string> CollectDaemonLogsAsync(DateTime testStartTime, string
             return daemonLog;
         }
 
-        public IEdgeDaemon CreateEdgeDaemon(Option<string> _, Option<string> bootstrapAgentImage, Option<Registry> bootstrapRegistry) => new EdgeDaemon(bootstrapAgentImage, bootstrapRegistry);
+        public IEdgeDaemon CreateEdgeDaemon(Option<string> _, Option<string> bootstrapAgentImage, Option<Registry> bootstrapRegistry) => EdgeDaemon.CreateAsync(bootstrapAgentImage, bootstrapRegistry).Result;
```

Comment:
This needs to become `CreateEdgeDaemonAsync`, instead of using `Task.Result`.

Reply:
This got refactored into using lazy. :D